docs/controllers/scaling.md

```diff
@@ -0,0 +1,73 @@
+# Scaling
+
+This chapter is about strategies for scaling controllers and the tradeoffs these strategies make.
+
+## Motivating Questions
+
+- Why is the reconciler lagging? Are there too many resources being reconciled?
+- What happens when your controller starts managing resource sets so large that it starts significantly impacting your cpu or memory use?
```

Comment:
nit: eh, I'm not proud of myself for making such an insignificant comment, but it feels to me that we should maybe caps this to be consistent with other paragraphs in the rest of the doc.

```suggestion
- What happens when your controller starts managing resource sets so large that it starts significantly impacting your CPU or memory use?
```